cliquet/__init__.py

```diff
@@ -16,6 +16,13 @@
 from cliquet.logs import logger
 
 
+import warnings
+warnings.warn("Cliquet is now deprecated. Please update to kinto.core at "
```

Comment:
It seems that ``DeprecationWarning`` are ignored by default so we need to run ``python -Walways`` to see them.
Maybe we could use ``logger.warn`` instead or both?